### DISCUSSION TITLE
Essex spawns make no sense

### DISCUSSION THREAD

**Discussion by Therightroad2009** [SoftbearStudios/mk48#219] [General]
Everytime i am with a Lvl 6 or higher aprox. (most likely a fleet)  an Essex suddenly spawns in our face while we're low or a hard situation, we instantly waste all of our weapons on the Essex and then most of the times it will upgrade to Lvl 7 and kill all of us, this is annoying bc  this kind of trash spawn only happens on Essex which is extremely op against subs, when i die because of this i keep getting spawnkilled til lvl 1 and ragequit. WHY DOES IT HAPPEN ONLY ON ESSEX AND NOT OTHERS

**Comment by finnbear** [comment]
Essex is very long compared to its level. Long ships are difficult to spawn well and low level ships are common.

We hope to improve the situation in the future!

**Reply by Therightroad2009** [reply]
Thank you for the answer. However it doesn't happen in other long ships, only essex
